### PR TITLE
Added content related to global signout

### PR DESCRIPTION
**Pull request by royjit**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/auth/fragments/native_common/signout/common.md [hunk 1]
```diff
@@ -7,3 +7,9 @@ Calling signOut without any options will just delete the local cache and keychai
 
 <inline-fragment platform="android" src="~/lib/auth/fragments/android/signout/20_global_signout.md"></inline-fragment>
 <inline-fragment platform="ios" src="~/lib/auth/fragments/ios/signout/20_global_signout.md"></inline-fragment>
+
+Calling signout with `globalSignOut = true` will invalidate all the Cognito User Pool token of the signed in user. If the user is signed in into a device, they wont be authorized to perform task that require a valid token when a global signout is called from some other device.
```
Comment (TrekSoft) [inline docs/lib/auth/fragments/native_common/signout/common.md:11]: signed into a device

Comment (TrekSoft) [reply docs/lib/auth/fragments/native_common/signout/common.md:11]: perform a task that requires a valid token

Comment (royjit) [reply docs/lib/auth/fragments/native_common/signout/common.md]: Fixed

File: docs/lib/auth/fragments/native_common/signout/common.md [hunk 2]
```diff
@@ -7,3 +7,9 @@ Calling signOut without any options will just delete the local cache and keychai
 
 <inline-fragment platform="android" src="~/lib/auth/fragments/android/signout/20_global_signout.md"></inline-fragment>
 <inline-fragment platform="ios" src="~/lib/auth/fragments/ios/signout/20_global_signout.md"></inline-fragment>
+
+Calling signout with `globalSignOut = true` will invalidate all the Cognito User Pool token of the signed in user. If the user is signed into a device, they wont be authorized to perform a task that requires a valid token when a global signout is called from some other device.
```
Comment (palpatim) [inline docs/lib/auth/fragments/native_common/signout/common.md:11]: 1. s/all the Cognito User Pool token/all the Cognito User Pool tokens/
1. s/wont/won't/

Comment (royjit) [reply docs/lib/auth/fragments/native_common/signout/common.md]: Fixed

File: docs/lib/auth/fragments/native_common/signout/common.md [hunk 3]
```diff
@@ -7,3 +7,9 @@ Calling signOut without any options will just delete the local cache and keychai
 
 <inline-fragment platform="android" src="~/lib/auth/fragments/android/signout/20_global_signout.md"></inline-fragment>
 <inline-fragment platform="ios" src="~/lib/auth/fragments/ios/signout/20_global_signout.md"></inline-fragment>
+
+Calling signout with `globalSignOut = true` will invalidate all the Cognito User Pool tokens of the signed in user. If the user is signed into a device, they won't be authorized to perform a task that requires a valid token when a global signout is called from some other device.
+
+<amplify-callout>
```
Comment (jpignata) [inline docs/lib/auth/fragments/native_common/signout/common.md:13]: warning, maybe?

File: docs/lib/auth/fragments/native_common/signout/common.md [hunk 4]
```diff
@@ -7,3 +7,9 @@ Calling signOut without any options will just delete the local cache and keychai
 
 <inline-fragment platform="android" src="~/lib/auth/fragments/android/signout/20_global_signout.md"></inline-fragment>
 <inline-fragment platform="ios" src="~/lib/auth/fragments/ios/signout/20_global_signout.md"></inline-fragment>
+
+Calling signout with `globalSignOut = true` will invalidate all the Cognito User Pool tokens of the signed in user. If the user is signed into a device, they won't be authorized to perform a task that requires a valid token when a global signout is called from some other device.
```
Comment (jpignata) [inline docs/lib/auth/fragments/native_common/signout/common.md:11]: "and will need to login again?"

File: docs/lib/auth/fragments/native_common/signout/common.md [hunk 5]
```diff
@@ -7,3 +7,9 @@ Calling signOut without any options will just delete the local cache and keychai
...
+
+<amplify-callout>
+Global signout functionality does not work if you use one of the web UI sign in methods.
```
Comment (jpignata) [inline docs/lib/auth/fragments/native_common/signout/common.md:14]: Can these customers do something to globally sign out a user?

File: docs/lib/auth/fragments/native_common/signout/common.md (hunk 4 above)
Comment (jpignata) [inline docs/lib/auth/fragments/native_common/signout/common.md:11]: Is this a boolean for both platforms?

Comment (TrekSoft) [reply docs/lib/auth/fragments/native_common/signout/common.md:11]: Yep

File: docs/lib/auth/fragments/native_common/signout/common.md (hunk 5 above)
Comment (TrekSoft) [reply docs/lib/auth/fragments/native_common/signout/common.md:14]: Not yet - that's the bug we talked about during GA launch we want to address as a fast follow.

File: docs/lib/auth/fragments/native_common/signout/common.md (hunk 3 above)
Comment (royjit) [reply docs/lib/auth/fragments/native_common/signout/common.md]: Changed

File: docs/lib/auth/fragments/native_common/signout/common.md (hunk 4 above)
Comment (royjit) [reply docs/lib/auth/fragments/native_common/signout/common.md]: Added

**Pull request merged**